### DISCUSSION TITLE
Game compatibility

### DISCUSSION THREAD

**Discussion by Torrikusu** [skyline-emu/skyline#389] [Q&A] [answered]
Is there a list of games that works  or compatibility chart where to find games that work and don't work on the mean time of developing. Just curious

**Comment by vicvenpet** [comment, marked as answer]
Here:

https://docs.google.com/spreadsheets/d/1oFND6f0vcQKsb8dMiVuhrwY-_Tkzf2RyEbjBM64b_eM/edit#gid=0

**Reply by Fanatic4** [reply]
File has been deleted ?

**Reply by vicvenpet** [reply]
> Here are all the compatible games on [Skyline](https://github.com/skyline-emu/skyline):
> 
> [Skyline's Compatibility List](https://github.com/skyline-emu/skyline-games-list/issues)

**Comment by Brray07** [comment]
is there a new list?

**Reply by iniMerchant64** [reply]
No, why would that change in less than a month

**Comment by Wirufurido** [comment]
Compatibility list is not working anymore ? Any idea where to find another one ?

**Reply by vicvenpet** [reply]
Here are all the compatible games on [Skyline](https://github.com/skyline-emu/skyline):

[Skyline's Compatibility List](https://github.com/skyline-emu/skyline-games-list/issues)
